packages/augur-ui/src/modules/portfolio/components/portfolio-view/portfolio-view.tsx

```diff
@@ -52,6 +54,14 @@ export default class PortfolioView extends React.Component<
   render() {
     const s = this.state;
 
+    const theme = getTheme();
```

Comment:
convert this whole component into a pure component, drop getTheme in favor of the `const { theme } = useAppStatusStore()` model. This is all in betting-ui so you might need to merge the latest betting-ui branch. you can find the `useAppStatusStore()` function in `app/store/app-status`